setup.cfg

```diff
@@ -0,0 +1,12 @@
+[flake8]
+max-line-length = 88
+max-complexity = 6
+exclude = */migrations/*
+
+[tool:isort]
```

Comment:
Shouldn't it be just `[isort]` instead of `[tool:isort]`? According to isort config guide it should be the former one.